Libs/vtkSegmentationCore/vtkSegment.h

```diff
@@ -143,10 +137,9 @@ class vtkSegmentationCore_EXPORT vtkSegment : public vtkObject
   /// This is the default identifier of the segment within segmentation, so needs to be unique within a segmentation
   char* Name;
 
-  /// Default color
-  /// Called default because this is only used initially indicating the original color of the segment,
-  /// but then copies are made for display and this member variable has no effect on it afterwards.
-  double DefaultColor[3];
+  /// Color
+  /// The actual color the segment is shown in. Can be overridden in the display node for advanced purposes
```

Comment:
we could be a bit more specific than "advanced purposes", for example:

The actual color the segment is shown in. Can be overridden in the display node to allow displaying segment in selected views in a different color